app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
This is implemented correctly and follows the requirements. Good use of a base class for the vaccine-related errors.